### ISSUE TITLE
[ISSUE] Edit Rest Period Crash

### ISSUE THREAD

**Issue by davcpas1234** [VersCreativeUK/FACETS#41]
**Application Version**

v0.0.6-rc3

**Issue Description**

A crash is observed when attempting to edit a rest period via long click dialog. **This crash is not reflected when swiping to the right and selecting Edit**.

**Steps to Reproduce**

1. Go to 'Rest & Sleep Routine'
2. Click on 'Show All'
3. Long Click a Rest Item
4. Select 'Edit'
4. Application Crashes

**Expected Behaviour**

Edit activity is shown and no crash observed

**Screenshots (Optional)**

![41-issue](https://user-images.githubusercontent.com/3764587/52585528-f7a90a00-2e2c-11e9-8ab8-9ea4c9e03434.gif)

**Comment by davcpas1234** [comment]
Stacktrace:
```
java.lang.RuntimeException: Unable to start activity ComponentInfo{uk.ac.bournemouth.facets/uk.ac.bournemouth.facets.Activities.RestSleepWakeRoutine.EditRSWItemActivity}: java.lang.NullPointerException: Provided document path must not be null.
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2904)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2979)
        at android.app.ActivityThread.-wrap11(Unknown Source:0)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1643)
        at android.os.Handler.dispatchMessage(Handler.java:105)
        at android.os.Looper.loop(Looper.java:180)
        at android.app.ActivityThread.main(ActivityThread.java:6944)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:240)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:853)
     Caused by: java.lang.NullPointerException: Provided document path must not be null.
```

**Comment by davcpas1234** [comment]
Resolved

**Comment by davcpas1234** [comment]
Will be released in `v0.0.6-rc4`
